### ISSUE TITLE
Custom interpolation symbols support

### ISSUE THREAD

**Issue by jorenham** [jsmodules/angular-material-calendar#54]
Currently the default interpolation symbols `{{` and `}}` and hard-coded in the templates. This makes it that configuring custom ones (e.g. `$interpolateProvider.startSymbol('{$'); $interpolateProvider.endSymbol('$}');`) will break angular-material-calendar . 
Is there an easy way to work around this or will this be fixed?

**Comment by bradberger** [comment]
Good point. I think it would be possible to fix if the code determined the current start/end symbols via `$interpolateProvider` before parsing the template, and then replaced the `{{` and `}}` with whatever the values were if they were different.

Is that something you might be able to submit a PR for? I'm not going to be able to get to this very quickly. If you can't, then let's at least leave it open until myself or someone else has a chance to give it a go.
